.github/workflows/action.yaml

```diff
@@ -0,0 +1,42 @@
+name: Checks
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  setup:
+    name: Go tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Parse Go mod file
+        id: go_mod_parse
+        run: echo "::set-output name=go_mod_version::$(go mod edit -json | jq --raw-output .Go)"
+      - name: Setup specific Go version
+        uses: actions/setup-go@v2
```

Comment:
`setup-go` has released v3.1 which adds `go-version-file: 'path/to/go.mod'.
Removing the need to define the `go_mod_parse` step manually.
We should update.

Reply:
Done in: 27bef9725b995f04231e76bc71e6760361e4b7bb + [actions/runs/2355918259](https://github.com/djdv/go-filesystem-utils/actions/runs/2355918259)